### ISSUE TITLE
[dev.icinga.com #9023] Toggling feature...

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icingaweb2#1590]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/9023

**Created by jmeyer on 2015-04-09 07:29:42 +00:00**

Assignee: _elippmann_
Status: _Resolved_  (closed on _2015-05-18 14:10:03 +00:00_)
Target Version: _2.0.0-rc1_
Last Update: _2015-05-18 14:10:03 +00:00 (in Redmine)_

---

The user is not "toggling" a feature, but enabling or disabling it. There is even no reason to not show this as we should be able to interpret whether a checkbox has been checked or unchecked.

**Changesets**

_2015-05-15 15:01:28 +00:00_ by _aklimov_ 58434d97438a19aafa33eee3f942c67436fe8b63

    ToggleInstanceFeaturesCommandForm::onSuccess(): show a notification for each feature about whether it has been enabled or disabled
    
    fixes #9023

_2015-05-18 12:20:17 +00:00_ by _aklimov_ 4fbadaea2c2a7e998d017d318dbb65b1c3a5306a

    ToggleInstanceFeaturesCommandForm::createElements(): make form labels shorter
    
    refs #9023

_2015-05-18 12:51:22 +00:00_ by _aklimov_ 729fb9a4d16d5667910bbc59350b8c2eb0dcb8af

    ToggleInstanceFeaturesCommandForm::onSuccess(): notifications: show which feature has been enabled or disabled
    
    refs #9023

_2015-05-18 13:18:26 +00:00_ by _aklimov_ 0d48ddcc062f67cb2fb4d5e6eaf81bb06aaeb481

    ToggleObjectFeaturesCommandForm::onSuccess(): show a notification for each feature about whether it has been enabled or disabled
    
    refs #9023

_2015-05-18 14:07:28 +00:00_ by _aklimov_ 7caba93a28c972585bafaaa0fa7dc59da59758e3

    Replace `...' w/ `..'
    
    refs #9023

_2015-05-18 14:09:07 +00:00_ by _aklimov_ 52d1cb8c8fe96a9eff00e6017ac19bc87bb5e6b9

    Merge branch 'bugfix/Toggle-FeaturesCommandForm-make-notifications-more-verbose-9023'
    
    fixes #9023
